doc/releases/changelog-dev.md

```diff
@@ -473,6 +473,9 @@
   operators have a valid `pauli_rep` property.
   [(#5177)](https://github.com/PennyLaneAI/pennylane/pull/5177)
 
+* The matrix of an empty `PauliSentence` instance is now correct (all-zeros).
+  Further, matrices of empty `PauliWord` and `PauliSentence` instances can be turned to matrices now.
```

Comment:
This entry seems incorrect. We're not returning all zeros, just a 1x1 matrix.

Reply:
```
>>> PauliSentence({}).to_mat(wire_order=[0, 1])
array([[0., 0., 0., 0.],
       [0., 0., 0., 0.],
       [0., 0., 0., 0.],
       [0., 0., 0., 0.]])
```

Reply:
```
>>> PauliSentence({}).to_mat()
array([[0.]])
>>> PauliSentence({}).to_mat().shape
(1, 1)
```
If not all-zero matrix, how else would you call it?

Reply:
Ah yes I forgot about the case with wire order provided. Thanks!